### PR TITLE
Re-order docs for SC website

### PR DESCRIPTION
**Pull request by MUTOgen**
We use folders in `/docs` as a category for sidebar menu
So, we want this structure to better fit this TOC https://github.com/shakacode/react_on_rails/blob/master/SUMMARY.md

Also, `README` updated to remove gitbook link and use SC website

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1335)

### CODE REVIEW

**Comment by justin808** [comment]
@MUTOgen @hussainakram We need a PR to address the rubocop and the rspec failures.

@MUTOgen if your changes are doc only, let me know and I'll still merge this.

**Comment by justin808** [comment]
1. Need to update for CI. I think you just need to rebase on master.
2. Lots of broken links on the new place. We need to convert relative links in the code to absolute links.
3. Need to make it clear that the docs directory should be indicated as the docs-website with a readme indicating that
   1.  links might not work when viewed on github
   2. PRs for docs are encouraged
   3. _Maybe https://github.com/gregberge/loadable-components/tree/master/website could provide some inspiration._
4. Need something on the shakacode.com pages that encourages edits to the docs

**Comment by justin808** [comment]
@MUTOgen **IMPORTANT**: I'm working on this branch and I'll be rebasing it, so contact me before any future work on this branch.

**Comment by justin808** [comment]
@Judahmeek please pick up on this task.

File: docs/contributor-info/releasing.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 We're now releasing this as a combined ruby gem plus npm package. We will keep the version numbers in sync.
 
 ## Testing the Gem before Release from a Rails App
-See [Contributing](../../CONTRIBUTING.md)
+See [Contributing](https://github.com/shakacode/react_on_rails/tree/master/CONTRIBUTING.md)
```
Comment (justin808) [inline docs/contributor-info/releasing.md:6]: @Judahmeek we might want this as part of the regular docs published to shakacode.com

**Review by justin808 — COMMENTED** [review]
One tiny suggestion. 

The main thing is that structure does not have broken links on the new gatsby publishing.

File: docs/contributor-info/releasing.md (hunk 1 above)
Comment (Judahmeek) [reply docs/contributor-info/releasing.md:6]: I agree. I think we also might want to move Projects & Kudos into /testimonials

**Comment by Judahmeek** [comment]
[The Additional Reading section header](https://github.com/shakacode/sc-website/blob/master/src/styleguide/page-components/DocArticle/DocSidebar.tsx#L13) needs to be removed when this PR is merged.

**Pull request merged**